### ISSUE TITLE
JAWS emits a grid name  twice in Forms mode

### ISSUE THREAD

**Issue by Vladimir-Goncharov-a11y** [FreedomScientific/standards-support#837]
### Summary
When a user tabs  into a grid using Forms mode, JAWS emits the grid name twice.  
To reproduce, you need a grid made with ^lt;table role="grid" aria-label="Example"&gt;
I've uploaded a text file, rename an extension to .html and open it in Chrome.
[odan_257_grid_role2.txt](https://github.com/FreedomScientific/standards-support/files/15339997/odan_257_grid_role2.txt)
There is a text field  which has no relationship with the grid, but needed to set focus outside of the grid and switch JAWS to Forms mode.
Set focus to the first field. Switch JAWS to Forms mode automatically or manually (depending on your JAWS settings).
Confirm the current JAWS mode by pressing alt-del.
Press Tab key to jump into simplified Grid example.
 JAWS transcript:
> Example table
> Column 1, Row 2
> Example
> Head r1 c3 // this is the row header
> Col 1 // this is the column header
> Cell r1 c1 // The first cell content
> column 1 row 2 // Focused cell coordinates

The bug: grid label "Example" is emitted twice.

### Expected result
The  grid label should be emitted only once.

### Actual result
The grid label is incorrectly emitted twice.

### Additional Information
Reproducible with default JAWS settings
Please, use Forms mode to reproduce this bug,  not the Virtual PC Cursor mode.
The example grid is significantly simplified, no JS attached, just role="grid" and tabstop in the grid cell.

#### JAWS version and build number
JAWS version 2024.2405.48.400

 
#### Operating System and version
Microsoft Windows 11 Professional Version 23H2 (OS Build 22631.3447), System Type x64 

#### Browser and version:
Google Chrome Version 124.0.6367.207

**Comment by stevefaulkner** [comment]
can reproduce, [test case](https://cdpn.io/pen/debug/QWRNNax)

@BrettLewisVispero  this seems like a low priority issue.
